src/Razor/Microsoft.AspNetCore.Razor.Language/src/BoundAttributeDescriptorComparer.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    internal class BoundAttributeDescriptorComparer : IEqualityComparer<BoundAttributeDescriptor>
+    internal sealed class BoundAttributeDescriptorComparer : IEqualityComparer<BoundAttributeDescriptor>
```

Comment:
Seeing how much `sealed` shows up in @pranavkm's PRs, I propose renaming it to "the Pranav modifier".

Reply:
#pranavPoints 😄